src/lib/int.ts

```diff
@@ -215,12 +391,17 @@ class UInt64 extends CircuitValue {
     } else {
       let xMinusY = this.value.sub(y.value).seal();
       let yMinusX = xMinusY.neg();
-      let xMinusYFits = xMinusY
-        .rangeCheckHelper(UInt64.NUM_BITS)
-        .equals(xMinusY);
-      let yMinusXFits = yMinusX
-        .rangeCheckHelper(UInt64.NUM_BITS)
-        .equals(yMinusX);
+
+      let xMinusYFits = Gadgets.rangeCheckHelper(
+        UInt64.NUM_BITS,
+        xMinusY
+      ).equals(xMinusY);
```

Comment:
aah.. now I get why `rangeCheckHelper()` was exposed and not just `rangeCheck()` -- because here we are using it with `equals()` instead of `assertEquals()`

Maybe we can create two different gadgets for those cases?
```ts
rangeCheck(n: number, x: Field): void; // rangeCheckHelper + assertEquals
isDefinitelyInRange(n: number, x: Field): Bool; // rangeCheckHelper + equals
```

weird name `isDefinitelyInRange()` because a false result doesn't prove that it's not in the range. feel free to pick a better name.